js/map.js

```diff
@@ -113,4 +89,39 @@ const renderMarkers = (similarAds) => {
   });
 };
 
+map
+  .on('load', () => {
+    enablePage();
+    getData(
+      (ads) => {
+        renderMarkers(ads.slice(0, SIMILAR_AD_COUNT));
+      },
```

Comment:
Пока до 10 штук можно не обрезать, обрезаться они будут на следующем задании с фильтрацией + запись лучше изменить так, у тебя пока 1 функция, если появится больше то для читаемости лучше будет вынести отсюда
```suggestion
      renderMarkers(ads.slice(0, SIMILAR_AD_COUNT)),
```